specification/common-types/resource-management/v5/types.json

```diff
@@ -624,6 +624,74 @@
           "type": "string"
         }
       }
+    },
+    "UsageName": {
+      "type": "object",
+      "description": "Name of the usage metrics",
+      "properties": {
+        "value": {
+          "type": "string",
+          "description": "The name of the usage metric."
+        },
+        "localizedValue": {
+          "type": "string",
+          "description": "The localized name of the usage metric."
+        }
+      }
+    },
+    "Usage": {
+      "type": "object",
+      "description": "Resource usage and limit.",
+      "properties": {
+        "unit": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The unit of usage measurement."
+        },
+        "currentValue": {
+          "readOnly": true,
+          "type": "number",
+          "format": "double",
+          "description": "The current usage of the resource."
+        },
+        "limit": {
+          "readOnly": true,
+          "type": "number",
+          "format": "double",
+          "description": "The maximum permitted usage of the resource."
+        },
+        "name": {
+          "readOnly": true,
+          "$ref": "#/definitions/UsageName",
+          "description": "The name of the type of usage."
+        },
+        "nextResetTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Optional.The time that the metric's value will reset."
+        },
+        "quotaPeriod": {
+          "type": "string",
+          "description": "Optional. The quota period that determines the length of time between value resets."
+        }
+      }
+    },
+    "ListUsagesResult": {
+      "type": "object",
+      "properties": {
+        "value": {
```

Comment:
include "nextLink" property as well so the API can be paginated.  This will fix your lintdiff error as well

Reply:
This api is already implemented based on the agreed contract, we can take this as an enhancement. Code change is done as per this contract.